src/lib/Libutil/avltree.c

```diff
@@ -130,6 +130,8 @@ typedef struct {
 static pthread_once_t avl_init_once = PTHREAD_ONCE_INIT;
 static pthread_key_t avl_tls_key;
 
+#define MAX_AVLKEY_LEN 1024
```

Comment:
I think you're right.  I took a look at the new usages I'm adding, and I *think* a length of 100 should be enough.